patternfly/patternfly-elements
# fix: umd.min modules now import umd.min builds of PFE dependencies

## Component name

All components are affected.

## What has changed and why

The build scripts (gulpfile & rollup) have changed, so that the `.umd.min` formatted bundles will import the `.umd.min` formatted bundles of other PFElements that they depend on (such as PFElement).

### Testing instructions

1. Run tests

Or

1. Run a build and inspect the `.js`, `.min.js`, `.umd.js`, and `.umd.min.js` bundles of any given element, such as pfe-card.
2. Ensure that the `.js` bundle imports `pfelement.js`
3. Ensure that the `.min.js` bundle imports `pfelement.min.js`
4. Ensure that the `.umd.js` bundle imports `pfelement.umd` (that's right, no `.js`)
5. Ensure that the `.umd.min.js` bundle imports `pfelement.umd.min` (that's right, no `.js`)

### Ready-for-merge Checklist

Check off items as they are completed.  Feel free to delete items if they are not applicable.



### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
